modules/cnf-du-creating-a-machine-config-manifest.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-creating-a-machine-config-manifest_{context}"]
+
+= Creating a machine config manifest
+
+To enable workload partitioning, you need to provide a `MachineConfig` manifest during installation to configure CRI-O and kubelet to know about the workload types.
+
+The manifest, without the encoded file content, looks like this:
+
+[source,yaml]
```

Comment:
sugg: add a block title like ".Example machine config for workload partitioning"

I'm not sure if that's precisely correct.  Up to you.